rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-10-24
+nightly-2022-05-23
```

Comment:
if you could do it for the node repo :smile:

Reply:
Its already done for aleph-node repo